systemsdk/easy-log-bundle/1.10/config/packages/test/easy_log.yaml

```diff
@@ -0,0 +1,7 @@
+easy_log:
```

Comment:
Same here

Reply:
It is already sensible defaults (we usually don't need to log debug info)